### ISSUE TITLE
Change the hostname redirects to "Not Found" page

### ISSUE THREAD

**Issue by kstaniek** [csm-aut/csm#16]
After changing the hostname in Edit Host and saving the change it redirects to page displaying Not Found, however the change is accepted.

**Comment by csm-aut** [comment]
Yes, if you changed the host on the host dashboard page because the hostname is used to lookup the URL.

**Comment by kstaniek** [comment]
Are you considering it as a bug that needs to be fixed?
